cinder/Dockerfile

```diff
@@ -24,6 +24,7 @@ RUN pip3 install https://github.com/emc-openstack/storops/archive/v${storops_ver
 ARG naviseccli=NaviCLI-Linux-64-x86-en_US-7.33.9.1.55-1.x86_64.rpm
 RUN curl -sSOL https://github.com/emc-openstack/naviseccli/raw/master/${naviseccli}
 RUN yum localinstall -y ${naviseccli}
+RUN yum -y update-minimal --security --sec-severity=Important --sec-severity=Critical
```

Comment:
The Manila Dockerfile is also needed to apply this change, right?

Reply:
Make sense. Added the line to Manila's Dockerfile.